docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -257,12 +278,87 @@ remove_action( 'bigcommerce/template/product/archive', bigcommerce()->templates-
+
+1. Using either a child theme or child plugin, declare a function named `update_add_to_cart_message()` that accepts `$js_i18n_array` as an arguement.
+2. Within `update_add_to_cart_message()`, set the new value for the `['ajax_add_to_cart_success']` key located in the nested `['cart']` array to `'Item added to cart. <a href="/cart">View Cart!</a>'`. Note that this is an example and your store's cart might have a different slug than `/cart`.
+3. Return the filtered `$js_i18n_array` array using a return statement.
+
+Your `update_add_to_cart_message()` function will look like the following:
+
+```php
+function update_add_to_cart_message( $js_i18n_array ){
+   $js_i18n_array['cart']['ajax_add_to_cart_success'] = 'Item added to cart. <a href="/cart">View Cart!</a>';
+   return $js_i18n_array;
+};
+```
+
+Following the `update_add_to_cart_message` function, call the `add_filter()` function that is available through the WordPress Plugin API. `add_filter()` hooks a function or method to a specific filter action. Hook `update_add_to_cart_message()` to the `bigcommerce/js_localization` filter hook that is provided by the BC4WP plugin.
+
+```php
+add_filter( 'bigcommerce/js_localization', 'update_add_to_cart_message' );
+```
+
+Once the file is saved, WordPress will now localize the modified string when an item is added to a cart by doing the following:
+
+1. When the `bigcommerce/js_localization` filter hook is applied, `add_filter()` invokes `update_add_to_cart_message()` and `$js_i18n_array` is passed in as an argument.
+2. `$js_i18n_array` is filtered and the value for `ajax_add_to_cart_success` is updated to `'Ttem added to cart. <a href="/cart">View Cart!</a>'` before being returned on the following line by our return statement.
+3. `Scripts.js` displays this updated string when a user adds an item to their cart.
+
 ## Styling Checkout
+
 BigCommerce for WordPress offers two possible checkout experiences, depending on whether the WordPress site has an installed SSL certificate.
 
 If no SSL is detected, shoppers will be redirected to the BigCommerce checkout page, on the domain of the BigCommerce store. Because BigCommerce stores provide an SSL certificate by default, this ensures that the checkout page is always accessed on an encrypted domain. If you are using this checkout experience, visit our [Stencil documentation](https://developer.bigcommerce.com/stencil-docs/template-files/customize-stencil-checkout/optimized-one-page-checkout) to review all available options for styling the checkout page.
 
-If an SSL is detected, shoppers will visit an embedded version of the BigCommerce checkout page, on your WordPress domain. This has the advantage of providing a seamless experience for the shopper. 
+If an SSL is detected, shoppers will visit an embedded version of the BigCommerce checkout page, on your WordPress domain. This has the advantage of providing a seamless experience for the shopper.

```

Comment:
If an SSL is detected, shoppers will visit an embedded version of the BigCommerce checkout page, on your WordPress domain. This (redirect) has the advantage of providing a seamless experience for the shopper.

Reply:
I am not sure if redirect is the right word, but I wanted to clarify what "this" is referring to.